zerver/forms.py

```diff
@@ -115,6 +115,7 @@ class RegistrationForm(forms.Form):
     realm_type = forms.IntegerField(required=False)
     is_demo_organization = forms.BooleanField(required=False)
     enable_marketing_emails = forms.BooleanField(required=False)
+    email_address_visibility = forms.IntegerField(required=False)
```

Comment:
I think we need to somewhere verify that the value they selected is a valid `email_address_visibility` value; Django often likes to do that in the forms system, but I'm not sure specifically what the best way to do that is here.

Reply:
We can use `TypedChoiceFields` here. I have used that here for now. Other way can be to just add a function similar to `clean_full_name` which will just check whether the values are valid and then return the value as it is if they are valid and raise an error otherwise.